CHANGELOG.md

```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v7.0.1](https://github.com/voxpupuli/puppet-php/tree/v7.0.1) (2020-01-21)
```

Comment:
@dhoppe ah we should probably regenerate this?

Reply:
I will take care of it.